Sources/AppcuesKit/Data/Models/Experiment.swift

```diff
@@ -23,5 +25,7 @@ extension Experiment: Decodable {
         case group
         case experimentID = "experimentId"
         case experienceID = "experienceId"
+        case goalID = "goalId"
+        case contentType
```

Comment:
the JSON keys are `goal_id` and `content_type` in the response, but appear as camelCase to us, due to https://github.com/appcues/appcues-ios-sdk/blob/main/Sources/AppcuesKit/Data/Networking/NetworkClient.swift#L124